lib/Schema.js

```diff
@@ -87,13 +87,24 @@ function Schema(obj, options) {
 Schema.prototype.toDynamo = function(model) {
 
   var dynamoObj = {};
-  //debug('toDynamo with schema attributes', this.attributes);
-  for(var name in this.attributes) {
+
+  for(var name in model) {
+    if(!model.hasOwnProperty(name)){
+      continue;
+    }
```

Comment:
Needed to add this continue in order to avoid trying to turn schema, options, ..., etcetera into `Attribute` objects that would attempt to be persisted.